book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
```

Comment:
`#add_rental` of `Person` class should receive `book, date` as parameters.
```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
Fixed